BUILDING.md

```diff
@@ -111,7 +111,7 @@ to use `go mod` command to modify the dependencies. After modifition, you should
 and `go mod vendor` to ensure the `go.mod`, `go.sum` files and `vendor` directory are up to date.
 Changes to these files should become a single commit for a PR which relies on vendored updates.
 
-Please refer to [RUNC.md](/RUNC.md) for the currently supported version of `runc` that is used by containerd.
+Please refer to [RUNC.md](/docs/RUNC.md) for the currently supported version of `runc` that is used by containerd.
```

Comment:
maybe `./docs/RUNC.md` ?

Reply:
There is no strict hierarchical relationship between `BUNDLING.md` and `RUNC.md`. I feel it is better to use absolute paths

And the absolute paths ensure that the `/docs/RUNC.md` link remains active after the `BUNDLING.md` has been moved, unless the `/docs/RUNC.md` has also been moved

`/docs/RUNC.md` is also used [elsewhere](https://github.com/containerd/containerd/search?q=RUNC.md&type=)